src/components/site-form.tsx

```diff
@@ -480,9 +492,34 @@ export const SiteForm = ( {
 											</div>
 										) }
 
-										{ setUseCustomDomain && setCustomDomain && (
+										{ useCustomDomain && setEnableHttps && (
+											<div className="flex items-center gap-2 mt-4">
+												<input
+													type="checkbox"
+													id="enable-https"
+													checked={ enableHttps }
+													onChange={ ( e ) => setEnableHttps( e.target.checked ) }
+												/>
+												<label htmlFor="enable-https">{ __( 'Enable HTTPS' ) }</label>
+											</div>
+										) }
+
+										{ useCustomDomain && setEnableHttps && (
 											<div className="text-a8c-gray-50 text-xs mt-2">
-												{ __( 'Your system password will be required to set up the domain.' ) }
+												{ __(
+													'You need to manually add the Studio certificate authority to your keychain and trust it.'
+												) }{ ' ' }
+												<Button
+													variant="link"
+													onClick={ () => {
+														getIpcApi().openURL(
+															'https://developer.wordpress.com/docs/developer-tools/studio/ssl-in-studio/'
+														);
+													} }
+												>
+													{ __( 'Learn how' ) }
+													<span aria-label={ __( '(opens in a new tab)' ) }>&#8599;</span>
```

Comment:
```suggestion
													<span aria-label={ __( '(opens in a web browser)' ) }>&#8599;</span>
```
I see we already have a few instances of this description in the code. "In a new tab" doesn't really make sense in a desktop app, so I would advise updating all instances of this string.